pkg/build/pipelines/python/import.yaml

```diff
@@ -21,12 +20,16 @@ inputs:
     description: |
       Commands to import packages, each line is a separate command. Example:
         from libfoo import bar
+        # test that otherthing can be imported from asdf
         from asdf import otherthing
-        import bark
+        import bark # this is like woof
+
+      full-line and inline comments are supported via '#'
     required: false
 
 pipeline:
   - runs: |
+      set +x
```

Comment:
Do we need/want this in all regular non-debug runs of this test?

Reply:
imo no. its not helpful, and spams WARN to the screen and makes good output harder to read.

note, this is `set +x` (turn off debugging) rather than turn it on.

Reply:
for reference, this is `set -x` (without this change):
```
2024/07/09 13:06:20 INFO running the main test pipeline
2024/07/09 13:06:20 INFO running step "Test a python package import, with optional from clause"
2024/07/09 13:06:20 WARN + export 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
2024/07/09 13:06:20 WARN + '[' -d /home/build ]
2024/07/09 13:06:20 WARN + cd /home/build
2024/07/09 13:06:20 WARN + exit 0
2024/07/09 13:06:20 WARN + export 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
2024/07/09 13:06:20 WARN + '[' -d /home/build ]
2024/07/09 13:06:20 WARN + cd /home/build
2024/07/09 13:06:20 WARN + PYTHON=DEFAULT
2024/07/09 13:06:20 WARN + SINGLE_IMPORT=
2024/07/09 13:06:20 WARN + MULTIPLE_IMPORTS='from codeowners import CodeOwners
2024/07/09 13:06:20 WARN '
2024/07/09 13:06:20 WARN + FROM_PKG=
2024/07/09 13:06:20 WARN + '[' DEFAULT '=' DEFAULT ]
2024/07/09 13:06:20 WARN + glob='/usr/bin/python3.[0-9][0-9]'
2024/07/09 13:06:20 WARN + n=0
2024/07/09 13:06:20 WARN + '[' -x /usr/bin/python3.12 ]
2024/07/09 13:06:20 WARN + n=1
2024/07/09 13:06:20 WARN + '[' 1 -ne 1 ]
2024/07/09 13:06:20 WARN + echo 'using python /usr/bin/python3.12'
2024/07/09 13:06:20 WARN + PYTHON=/usr/bin/python3.12
2024/07/09 13:06:20 WARN + '[' -n  ]
2024/07/09 13:06:20 WARN + '[' -n  ]
2024/07/09 13:06:20 WARN + fail_flag=0
2024/07/09 13:06:20 WARN + '[' -n 'from codeowners import CodeOwners
2024/07/09 13:06:20 WARN ' ]
2024/07/09 13:06:20 INFO using python /usr/bin/python3.12
2024/07/09 13:06:20 WARN + mktemp
2024/07/09 13:06:20 WARN + importf=/tmp/tmp.iR84Ga
2024/07/09 13:06:20 WARN + printf '%s\n' 'from codeowners import CodeOwners
2024/07/09 13:06:20 WARN '
2024/07/09 13:06:20 WARN + read line
2024/07/09 13:06:20 WARN + line='from codeowners import CodeOwners'
2024/07/09 13:06:20 WARN + set -f
2024/07/09 13:06:20 WARN + echo from codeowners import CodeOwners
2024/07/09 13:06:20 WARN + cmd='from codeowners import CodeOwners'
2024/07/09 13:06:20 WARN + '[' -z 'from codeowners import CodeOwners' ]
2024/07/09 13:06:20 WARN + perform_import 'from codeowners import CodeOwners'
2024/07/09 13:06:20 WARN + command='from codeowners import CodeOwners'
2024/07/09 13:06:20 WARN + /usr/bin/python3.12 -c 'from codeowners import CodeOwners'
2024/07/09 13:06:20 INFO /usr/bin/python3.12 -c "from codeowners import CodeOwners": PASS
2024/07/09 13:06:20 WARN + echo '/usr/bin/python3.12 -c "from codeowners import CodeOwners": PASS'
2024/07/09 13:06:20 WARN + read line
2024/07/09 13:06:20 WARN + line=
2024/07/09 13:06:20 WARN + set -f
2024/07/09 13:06:20 WARN + echo
2024/07/09 13:06:20 WARN + cmd=
2024/07/09 13:06:20 WARN + '[' -z  ]
2024/07/09 13:06:20 WARN + continue
2024/07/09 13:06:20 WARN + read line
2024/07/09 13:06:20 WARN + rm -f /tmp/tmp.iR84Ga
2024/07/09 13:06:20 WARN + exit 0
```

This is `set +x`
```
2024/07/09 13:06:39 INFO running the main test pipeline
2024/07/09 13:06:39 INFO running step "Test a python package import, with optional from clause"
2024/07/09 13:06:39 WARN + export 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
2024/07/09 13:06:39 WARN + '[' -d /home/build ]
2024/07/09 13:06:39 WARN + cd /home/build
2024/07/09 13:06:39 WARN + exit 0
2024/07/09 13:06:39 WARN + export 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
2024/07/09 13:06:39 WARN + '[' -d /home/build ]
2024/07/09 13:06:39 WARN + cd /home/build
2024/07/09 13:06:39 WARN + set +x
2024/07/09 13:06:39 INFO using python /usr/bin/python3.12
2024/07/09 13:06:39 INFO /usr/bin/python3.12 -c "from codeowners import CodeOwners": PASS
```

Reply:
I did notice that above, that is output of `make test/<package>` in wolfi/os.
It runs with `--debug`, so maybe i agree with @xnox and we should actually change the make target to not pass `--debug` by default.